apps/admin/integration-testing/package.json

```diff
@@ -25,8 +25,9 @@
     ]
   },
   "dependencies": {
-    "@playwright/test": "^1.37.1",
+    "@playwright/test": "^1.48.1",
```

Comment:
Why the bump here? Wondering if we should pick a version that sort of matches our kiosk-browser electron version.

And while you're here, might be worth making all the Playwright versions consistent across the repo, so we don't have multiple chromium installations floating around - I'm seeing `1.40.1` in our printing packages

Reply:
It was get to access to the `clock` API. The monorepo validation script reminded me to update it elsewhere, but not in libs/printing or design, so I'll update those and look into why monorepo didn't check there.

Reply:
Ah, makes sense it wouldn't catch those, since those are importing the general `plapywright` package